.changeset/hot-steaks-buy.md

```diff
@@ -0,0 +1,7 @@
+---
+"@directus/api": patch
+"@directus/sdk-schema-generator": major
+"@directus/types": patch
+---
+
+Added TypeScript Schema Generator for the Directus SDK
```

Comment:
To make it a bit clearer that it is a new feature:

```suggestion
Introduce TypeScript Schema Generator for the Directus SDK
```

or

```suggestion
Added new TypeScript Schema Generator for the Directus SDK
```

Reply:
Doesnt "Added" already imply that something that didnt exist before was introduced? else it would be "Updated" or "Fixed".